<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindow</class>
 <widget class="QDialog" name="SearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>120</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>PT Mono</family>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Find/Replace</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="inputLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="findLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Find: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="findText"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="replaceLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QCheckBox" name="replaceCheckBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Replace:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="replaceText"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="paramsLayout">
     <item>
      <widget class="QComboBox" name="findModeComboBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentText">
        <string>Find Backward</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Find Backward</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Find Case Sensitively</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Find Whole Words</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="regexCheckBox">
       <property name="text">
        <string>RegEx</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
